### PR TITLE
 Add new script command: translate tools

### PR DESCRIPTION
**Pull request by tiancheng92**
## Description

* new script command:
 * commands/dictionary/translate-to-en.sh: 
  * translate to english, show dictionary and copy brief translation to clipboard
 * commands/dictionary/translate-to-en.sh: 
  * translate to chinese, show dictionary and copy brief translation to clipboard

## Type of change

- [x] New script command

## Dependencies / Requirements

* Dependency: requires translate-shell
* Install with Homebrew: `brew install translate-shell`

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/dictionary/translate-to-en.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# @raycast.packageName Dictionary
+
+# Optional parameters:
+# @raycast.argument1 {"type": "text", "placeholder": "Word or Sentence"}
+# @raycast.icon 📖
+
+# Documentation:
+# @raycast.author tiancheng92
+# @raycast.authorURL https://github.com/tiancheng92
+# @raycast.description translate and copy brief translation to clipboard
```
Comment (unnamedd) [inline commands/dictionary/translate-to-en.sh:19]:
```suggestion
# @raycast.description Translate and copy brief translation to clipboard
```

File: commands/dictionary/translate-to-zh.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# @raycast.packageName Dictionary
+
+# Optional parameters:
+# @raycast.argument1 {"type": "text", "placeholder": "Word or Sentence"}
+# @raycast.icon 📖
+
+# Documentation:
+# @raycast.author tiancheng92
+# @raycast.authorURL https://github.com/tiancheng92
+# @raycast.description translate and copy brief translation to clipboard
```
Comment (unnamedd) [inline commands/dictionary/translate-to-zh.sh:19]:
```suggestion
# @raycast.description Translate and copy brief translation to clipboard
```

File: commands/dictionary/translate-to-zh.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Dependency: requires translate-shell
+# Install with Homebrew: `brew install translate-shell`
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Translate to ZH
+# @raycast.mode fullOutput
+# @raycast.packageName Dictionary
```
Comment (unnamedd) [inline commands/dictionary/translate-to-zh.sh:10]:
```suggestion
# @raycast.packageName Translate Shell
```

File: commands/dictionary/translate-to-en.sh [hunk 4]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Dependency: requires translate-shell
+# Install with Homebrew: `brew install translate-shell`
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Translate to EN
+# @raycast.mode fullOutput
+# @raycast.packageName Dictionary
```
Comment (unnamedd) [inline commands/dictionary/translate-to-en.sh:10]:
```suggestion
# @raycast.packageName Translate Shell
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
As your Script Command isn’t related exactly to a `Dictionary` I strongly suggest you to move them to a new folder called `translate-shell` inside of the `apps` folder instead of the folder. `dictionary` originally created by you. Inside of the `apps` is where we are organizing all the Script Commands related to some specific app.

**Comment by tiancheng92** [comment]
> As your Script Command isn’t related exactly to a `Dictionary` I strongly suggest you to move them to a new folder called `translate-shell` inside of the `apps` folder instead of the folder. `dictionary` originally created by you. Inside of the `apps` is where we are organizing all the Script Commands related to some specific app.

I have moved the folder.

**Pull request merged**